app/src/main/java/com/keylesspalace/tusky/fragment/ViewVideoFragment.kt

```diff
@@ -95,6 +94,15 @@ class ViewVideoFragment : ViewMediaFragment(), Injectable {
 
     private lateinit var mediaSourceFactory: DefaultMediaSourceFactory
 
+    /** Have we received at least one "READY" event? */
+    private var haveStarted = false
+
+    /** Is there a pending autohide? (We can't rely on Android's tracking because that clears on suspend.) */
+    private var pendingHideToolbar = false
+
+    /** Prevent the next play start from queueing a toolbar hide. */
+    private var suppressNextHideToolbar = false
```

Comment:
`pendingHideToolbar` and `suppressNextHideToolbar` are flags set at a certain point in the state machine in order to get the behavior of "if you intentionally tap to display the controls, and then leave and re-enter the app, the controls autohide does not engage after four seconds". I don't like the "spooky action at a distance" nature of `suppressNextHideToolbar` but since the events media3 gives us are limited I think this is the only way to detect the relevant case was hit. If we decide the controls should *always* hide after four seconds even after a tap-to-show-then-suspend-then-resume, we can remove these two variables and their associated logic.